lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -92,3 +103,28 @@
         }
     }
 }
+
+.app-metadata-tab .adf-metadata-properties .mat-expansion-panel {
+    border: 1px solid var(--adf-metadata-property-panel-border-color);
+    border-radius: 12px;
+    margin-bottom: 12px;
+}
+
+.acs-details-container {
```

Comment:
This should not be here probably. We are in ADF project so we should not have here referring to classes from projects which depends on ADF like ACA, ADW etc. Remember that ADF can be used by customer as it is open source so it should work without classes from ACA/ADW. Please write these styles properly

Reply:
These classes have been moved to this PR : https://github.com/Alfresco/alfresco-content-app/pull/3414/files